posthog-web/README.md

```diff
@@ -22,6 +22,7 @@ import PostHog from 'posthog-js-lite'
 const posthog = new PostHog('my-api-key', {
   /* options, e.g. for self-hosted users */
   // host: "https://my-posthog.app.com"
+  // trackHistoryEvents: true // Enable tracking of History API navigation events as pageviews
```

Comment:
> trackHistoryEvents is not a property that exists in our main posthog-js library, and I'm afraid I wouldn't want this library to have more features than our main lib.

i'm ok with adding this here if we then also add it in posthog-js... it's a super cool improvement

(we should definitely play with it running agains posthog locally since that's a SPA to figure out if it does anything weird but that's only me being cautious :))

---

in the main lib we tend to call config options `captureX` so might be worth keeping that here and calling it `captureHistoryEvents` seems nit-picky but the cognitive overload of config grows over time :)

Reply:
Will apply the changes, I've bundled this version into a repo to give it a few shots as well: https://lricoy.github.io/test-posthog-history/

Reply:
```suggestion
  // captureHistoryEvents: true // Enable tracking of History API navigation events as pageviews
```